apps/client/src/app/other-resources/other-resources-result/other-resources-result.component.ts

```diff
@@ -0,0 +1,212 @@
+import { Component, EventEmitter, OnInit } from '@angular/core';
+import { FhirService } from '../../shared/fhir.service';
+import { ActivatedRoute } from '@angular/router';
+import { DomainResource, OperationOutcome } from '../../../../../../libs/tof-lib/src/lib/stu3/fhir';
+import { getErrorString } from '../../../../../../libs/tof-lib/src/lib/helper';
+import { NgbModal, NgbTabChangeEvent } from '@ng-bootstrap/ng-bootstrap';
+import {saveAs} from 'file-saver';
+import { ChangeResourceIdModalComponent } from '../../modals/change-resource-id-modal/change-resource-id-modal.component';
+import { ConfigService } from '../../shared/config.service';
+import {Globals} from '../../../../../../libs/tof-lib/src/lib/globals';
+import { ValidatorResponse } from 'fhir/validator';
+
+class OpenedResource {
```

Comment:
The OpenedResource class was a concept that was needed when multiple resources could be opened on the same screen. I'm not so sure that is needed now that the screen only shows one resource. I think these properties can probably just be specified directly on the OtherResourcesResultComponent class...

Reply:
I'll modify this to not have the OpenedResource class.  Shouldn't be a problem.